bin/rportal

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+
+commands = {
+    "up": "docker-compose --env-file ./docker-compose.env up",
+    "down": "docker-compose --env-file ./docker-compose.env down",
+    "build": "docker-compose --env-file ./docker-compose.env build",
+    "rebuild-index": "docker-compose --env-file ./docker-compose.env run --rm api ./manage.py search_index --rebuild",
+    "populate-db": "docker-compose --env-file ./docker-compose.env run --rm api python3 manage.py populate_test_database",
+    "recreate-schema": "./recreate_schema.sh",
+    "test-api": "docker-compose run --env-file ./docker-compose.env --rm api ./run_tests.sh",
```

Comment:
I've been using:
```
docker-compose run --rm api bash -c "python3 wait_for_elasticsearch.py && ./manage.py makemigrations resources_portal"
```
to create my migrations, because I think a lot of API commands expect elasticsearch to be up. I believe that, `rebuild-index`, `populate-db`, and `test-api` all would benefit from waiting for elasticsearch to come up.

Additionally, could you add a `makemigrations` command to this?